src/inference/main.py

```diff
@@ -0,0 +1,47 @@
+import os
+from dotenv import load_dotenv
+from INCLUDE.evaluate import inference, get_inference_args
+from fastapi import FastAPI
+from azure.storage.blob import BlobServiceClient
+
+
+# Load environment variables
+load_dotenv()
```

Comment:
Does it make sense to add the .env template file so we know what environment variables to pass?

Reply:
ah thanks, I was going to do that but forgot :D Let me add the sample env

Reply:
added :)